app/src/main/java/org/mozilla/focus/utils/publicsuffix/PublicSuffixPatterns.java

```diff
@@ -41,7 +37,9 @@
         } catch (IOException e) {
             throw new IllegalStateException("resource publicsuffixlist could not be opened but is bundled with app", e);
         } finally {
-            IOUtils.safeStreamClose(reader);
+            try {
+                if (reader != null) { reader.close(); }
```

Comment:
ugh java

I was going to say I remember this way to read streams and close them after using...

Reply:
We could add a safe stream close ext fn but I figured when in kotlin, we'd just close streams and let the app crash if they can't be closed because something really funky is going on.